lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -548,7 +548,16 @@ func (d *zfs) CreateVolumeFromCopy(vol Volume, srcVol Volume, copySnapshots bool
 
 		// Handle transferring snapshots.
 		if len(snapshots) > 0 {
-			sender = exec.Command("zfs", "send", "-R", srcSnapshot)
+			args := []string{"send", "-R"}
+
+			// Use raw flag if possible.
```

Comment:
Can you make this `Use raw flag is supported, this is required to send/receive encrypted volumes (and enables compression).`

Reply:
Done